src/server/responses.py

```diff
@@ -107,9 +73,27 @@ def could_not_verify() -> Response:
     )
 
 
-def internal_database_error(message: Message) -> Response:
+def _internal_database_error(message: Message) -> Response:
     """500: DatabaseProvider returned message code other than OK."""
     return make_response(
         {"message": f"Internal error: {message}"},
         500,
     )
```

Comment:
One more thing - I think we should just respond with `Internal Server Error`. I agree that knowing what exactly went wrong on our side would come in handy, but I would rather set up come kind of logger on server only (definitely work for another pr) to hide this info from client.

Reply:
Should I just leave this argument unused for now? Or remove it from the calls completely?

Reply:
Up2u, just don`t send `DatabaseProvider ...` message to client.